zh_cn/deco.metalltreppe.json

```diff
@@ -1,36 +0,0 @@
-			"8": [{"name":"futurepack:erz_blocke", "meta":4},{"name":"futurepack:color_iron", "meta":0},{"name":"futurepack:color_iron", "meta":1},{"name":"futurepack:color_iron", "meta":2},{"name":"futurepack:color_iron", "meta":3},{"name":"futurepack:color_iron", "meta":4},{"name":"futurepack:color_iron", "meta":5},{"name":"futurepack:color_iron", "meta":6},{"name":"futurepack:color_iron", "meta":7},{"name":"futurepack:color_iron", "meta":8},{"name":"futurepack:color_iron", "meta":9},{"name":"futurepack:color_iron", "meta":10},{"name":"futurepack:color_iron", "meta":11},{"name":"futurepack:color_iron", "meta":12},{"name":"futurepack:color_iron", "meta":13},{"name":"futurepack:color_iron", "meta":14},{"name":"futurepack:color_iron", "meta":15}],
-			"9": [{"name":"futurepack:erz_blocke", "meta":4},{"name":"futurepack:color_iron", "meta":0},{"name":"futurepack:color_iron", "meta":1},{"name":"futurepack:color_iron", "meta":2},{"name":"futurepack:color_iron", "meta":3},{"name":"futurepack:color_iron", "meta":4},{"name":"futurepack:color_iron", "meta":5},{"name":"futurepack:color_iron", "meta":6},{"name":"futurepack:color_iron", "meta":7},{"name":"futurepack:color_iron", "meta":8},{"name":"futurepack:color_iron", "meta":9},{"name":"futurepack:color_iron", "meta":10},{"name":"futurepack:color_iron", "meta":11},{"name":"futurepack:color_iron", "meta":12},{"name":"futurepack:color_iron", "meta":13},{"name":"futurepack:color_iron", "meta":14},{"name":"futurepack:color_iron", "meta":15}],
-			"out": [{"name":"futurepack:metal_treppe", "meta":0, "size":4},{"name":"futurepack:color_iron_stair0", "size":4},{"name":"futurepack:color_iron_stair1", "size":4},{"name":"futurepack:color_iron_stair2", "size":4},{"name":"futurepack:color_iron_stair3", "size":4},{"name":"futurepack:color_iron_stair4", "size":4},{"name":"futurepack:color_iron_stair5", "size":4},{"name":"futurepack:color_iron_stair6", "size":4},{"name":"futurepack:color_iron_stair7", "size":4},{"name":"futurepack:color_iron_stair8", "size":4},{"name":"futurepack:color_iron_stair9", "size":4},{"name":"futurepack:color_iron_stair10", "size":4},{"name":"futurepack:color_iron_stair11", "size":4},{"name":"futurepack:color_iron_stair12", "size":4},{"name":"futurepack:color_iron_stair13", "size":4},{"name":"futurepack:color_iron_stair14", "size":4},{"name":"futurepack:color_iron_stair15", "size":4}]
-		}
-	},
-	"信息：可使用喷枪进行染色"
-]
```

Comment:
why was this file deleted?

Reply:
They have no practical effect.
There is no corresponding file in folder [en_us](https://github.com/Wugand/FuturePack-Language/tree/master/en_us)(so does [de_de](https://github.com/Wugand/FuturePack-Language/tree/master/de_de)).
This recipe is repeated with [deco.metall.json](https://github.com/Wugand/FuturePack-Language/blob/5cd98289a5107e4f97287449126d9d33e387225f/zh_cn/deco.metall.json#L6)（`		{"link":"futurepack:lang/part/cr_deco.metalltreppe1.json"},` ）.